connector/src/main/java/org/jboss/as/connector/subsystems/datasources/JdbcDriverAdd.java

```diff
@@ -97,14 +97,6 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final ModuleIdentifier moduleId;
         final Module module;
         String slot = operation.hasDefined(MODULE_SLOT.getName()) ? MODULE_SLOT.resolveModelAttribute(context, operation).asString() : null;
```

Comment:
This isn't a problem with this patch, it's a preexisting problem with the class.

This method is using the "operation" param in lots of places where it should use the "model" param, i.e. in call to resolveModelAttribute.

The way these classes work is populateModel is responsible for taking data from the operation and storing it in the model. This storage may involve some sort of transformation, e.g. converting node of ModelType.STRING with value ${foo} to a node of ModelType.EXPRESSION. Thereafter, the model is the authoritative source of data, and performRuntime should use it. The 'operation' param is just for stuff that can't be gotten from the model. Formerly the biggest thing was the resource address, but now in the typically cases that can be obtained more efficiently by a call to OperationContext.getCurrentAddress().